libraries/core-react/azure-pipelines.yml

```diff
@@ -47,10 +47,10 @@ steps:
       install
     displayName: 'Install dependencies'
 
-  - script: pnpm --filter ./libraries/core-react test
+  - script: pnpm --filter ./libraries/eds-core-react test
     displayName: 'Run tests'
 
   - script: >-
       ./node_modules/eslint/bin/eslint.js
-      ./libraries/core-react
+      ./libraries/eds-core-react
```

Comment:
missed revert name changes?

Reply:
Yup, that’s the problem with reverting this manually – and also why it will probably not be renamed again 😉